src/inline-svg.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable, Renderer2, RendererFactory2 } from '@angular/core';
+import { InlineSVGDirective } from './inline-svg.directive';
+
+@Injectable()
+export class InlineSVGService {
```

Comment:
Nitpick: do you mind indenting this file with 2 spaces like the other files?

Reply:
Sure, sorry. Seems like I had my editorconfig plugin deactivated.

Reply:
No worries, I'll fix it after merging it.